packages/wasi/src/polyfills/bigint.ts

```diff
@@ -1,8 +1,13 @@
 // A very simple workaround for Big int. Works in conjunction with our custom
 // Dataview workaround at ./dataview.ts
 
-const globalObj = typeof globalThis !== "undefined" ? globalThis : {};
+const globalObj: any =
+  typeof globalThis !== "undefined"
```

Comment:
Won't this still error because there is not variable called "globalThis" in Node? 

But I saw your commit message saying it fixes it so this is more of a question for me 😂

Reply:
You can action `typeof xxx` on anything... even on a variable that doesn't exist, and it will not fail! :)

Reply:
🤦‍♂